src/radius/nss/libnss-radius/nss_radius_common.c

```diff
@@ -167,6 +168,109 @@ static void init_rnm(RADIUS_NSS_CONF_B * conf) {
 
 }
 
+static int user_add(const char* cmd, const char* name, char* gid, char* sec_grp, char* gecos, 
```

Comment:
What is the purpose of accepting cmd as first argument if you have one function for each command (user_add, user_del, user_mod)?